How to/Library Bounds to Viewer Bounds/PDFViewerSample/PDFViewerSample/Pages/Index.cshtml

```diff
@@ -0,0 +1,88 @@
+﻿@page "{handler?}"
+@model IndexModel
+@{
+    ViewData["Title"] = "Home page";
+}
+
+<div class="text-center">
+    <ejs-pdfviewer id="pdfviewer"
+                   style="height:600px"
+                   documentPath="https://cdn.syncfusion.com/content/pdf/pdf-succinctly.pdf"
+                   serviceUrl="https://services.syncfusion.com/js/production/api/pdfviewer">
```

Comment:
@SF4524LogeshKumar  - Why we have used production service URL in ASP.NET core sample. Please check it and also PDF Viewer server library package was missed in the project reference.

Reply:
Addressed changes